components/admin-panel/sections/NotificationsSettings/ActivitySwitch.js

```diff
@@ -68,14 +79,31 @@ const ActivitySwitch = ({ account, activityType }) => {
   };
 
   return (
-    <InputSwitch
-      name={`${activityType}-switch`}
-      checked={isSubscribed}
-      disabled={isIndeterminate || isOverridedByAll}
-      onChange={event =>
-        handleToggle({ type: activityType, account: { id: account.id }, active: event.target.checked })
-      }
-    />
+    <Flex alignItems="center">
+      {isResetingSettings?.length > 0 ? (
+        <StyledTooltip
+          content={() => (
+            <FormattedMessage
+              id="NotificationsSettings.ToggleResetSettings"
+              defaultMessage="By toggling this setting, you're also reseting previously set options for: {activities}"
+              values={{ activities: isResetingSettings.join(', ') }}
```

Comment:
Maybe we can translate that using https://github.com/opencollective/opencollective-frontend/blob/56e4df659d3636656be7bc359fa442aa270ab2a3/lib/i18n/activities-classes.js?

Reply:
Already doing that when declaring `isResetingSettings`.